operations/deployment/terraform/modules/02_networking.tf

```diff
@@ -104,7 +103,7 @@ resource "aws_security_group" "allow_https" {
 resource "aws_security_group" "allow_ssh" {
  name        = "allow_ssh"
  description = "Allow SSH traffic"
- vpc_id      = aws_vpc.main.id
+ vpc_id      = var.create_vpc == "true" ? aws_vpc.main[0].id : null
```

Comment:
If the user is managing VPC outside of this GH Action, we'll need to support specifying the custom VPC ID for the security group and other resources.

Reply:
yep! https://github.com/bitovi/github-actions-deploy-stackstorm/issues/4

want to add a readme and publish to GHA marketplace. @LeoDiazL is working on the `vpc_id` option

Reply:
👍